ml-agents/mlagents/trainers/simple_env_manager.py

```diff
@@ -67,7 +67,7 @@ def _take_step(self, last_step: EnvironmentStep) -> Dict[AgentGroup, ActionInfo]
         all_action_info: Dict[str, ActionInfo] = {}
         for brain_name, step_info in last_step.current_all_step_result.items():
             all_action_info[brain_name] = self.policies[brain_name].get_action(
-                step_info
+                step_info, 0
```

Comment:
A little comment onto why this is 0? or make the argument 0 by default and explain why in the get_action method comments.

Reply:
Added comment - the argument is 0 by default but I made it explicit since it's explicit in the EnvironmentStep creation in this class.

Reply:
Also somewhat unrelated: the simple_env_manager is literally only used for the simple_rl test and nowhere else. Should we kill it at some point, or keep it around as an example?